packages/medusa/src/services/__tests__/event-bus.js

```diff
@@ -2,6 +2,7 @@ import Bull from "bull"
 import { MockManager, MockRepository } from "medusa-test-utils"
```

Comment:
suggestion: Since we are rewriting that file, I would suggest moving it to ts at the same time. I would also suggest to use the fixture and container approach that you can find in some other file that I have cleaned up